modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,402 @@
+= Anypoint Runtime Fabric Installation Prerequisites
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: install-port-reqs.adoc, install-sys-reqs.adoc
+
+Review the following requirements with your operations and IT teams to ensure that all prerequisites are met before installing Anypoint Runtime Fabric. The Runtime Fabric installation process fails if the minimum requirements are not met.
+
+If needed, contact your MuleSoft representative for assistance.
+ 
+See xref:architecture.adoc[Runtime Fabric Architecture] for a description of development and production configurations.
+
+== General Knowledge
+Knowledge in the following areas is required to perform a Runtime Fabric installation:
+
+* Linux system commands
+* Working in Linux with root access. Installation requires root access (permission to run the `sudo` command) on all servers in your environment
+* If running on Amazon Web Services (AWS), working with Terraform
+* Remote connection mechanisms, such as SSL and SSH
+
+== General Requirements
+Ensure that you plan for:
+
+* Creating separate Runtime Fabrics for production and non-production applications
+* Creating separate Runtime Fabrics based upon environment network scope, such as DMZ versus private network 
+* Ensuring that the Runtime Fabric default pod and services CIDR block do not interfere with the regions network
+
+== Infrastructure Requirements
+Perform the following actions:
```

Comment:
This makes me a little uneasy because it sounds like a task, but I am probably being overly sensitive.
"Ensure that the following conditions exist:"?

Reply:
So, I have already the first 2/3 of this section in my previous edit. 
I kind of hate to repeat that effort, so please refer to that previous review.
I can resend it it you like.

Reply:
Marking up just a few while I scan for where I left off....